pom.xml

```diff
@@ -76,6 +124,32 @@
 				<version>2.3.1</version>
 				<executions>
 					<execution>
+						<id>generate-rest-client</id>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+						<configuration>
+							<inputSpec>${project.basedir}/src/main/resources/geodb-swagger.yaml</inputSpec>
+							<language>java</language>
+							<library>okhttp-gson</library>
```

Comment:
You are missing a few things here. Maybe I'll update incubator docs to be more clear on the fact that you do not want swagger to generate a client class for you... So there is a config called interfaceOnly which will generate only the interface for the class and you're supposed to use retrofit and it's builder to inject the interface into the service wrapper class which calls the service. 

Additionally, our library isn't okhttp-gson, it's retrofit.... language part is correct.

Reply:
I was wondering if this is correct, the incubator wasn't very clear. It said I should use OKHttp and Retrofit libraries, so I figured I could use either one. I tried using both, one for client generation and the other for server generation, but couldn't get that to work yet, so I ended up with this setup. I know about the interfaceOnly part, I used that for the server generation config. Yes, if you could provide me with some docs that would be most helpful, like I said, all I had to go on is the incubator pdf which is not very clear.